service-commands/scripts/setup-instance.sh

```diff
@@ -103,7 +103,7 @@ case "$service" in
 
 		wait_for_instance $provider $user $name
 
-		execute_with_ssh $provider $user $name "bash ~/audius-protocol/service-commands/scripts/set-git-refs $audius_protocol_git_ref $audius_client_git_ref"
+        execute_with_ssh $provider $user $name "bash ~/audius-protocol/service-commands/scripts/set-git-refs.sh $audius_protocol_git_ref $audius_client_git_ref"
```

Comment:
What's the stance at Audius regarding tab/whitespace conversions and standards?

Reply:
I like whitespace

Reply:
personally prefer spaces but also please start this flame war in slack immediately

Reply:
lol! 😅 👨🏽‍🚒

Yeah, I prefer spaces as well! Ah, it looked like the commit dropped the tabs and changed them to white spaces on my side.

But ok, sounds good! I'll make sure my editor is set to white spaces, just wanted to make sure as I'll probably be messing with my editor settings next week as I start introducing PRs starting tomorrow.

I've been on different teams that like to keep convention, slowly change based on the standard at the time, or have a mix depending on the codebase. Didn't mean to start a potential flame war! 🤓

Reply:
hahaha i'm just kidding about the flame war, i'm not actually sure where team members stand on this, i'm p sure we standardized to spaces